content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -502,58 +502,13 @@ it.skip('returns "fizz" when number is multiple of 3', () => {
 
 ### Test Configuration
 
-To apply a specific Cypress [configuration](/guides/references/configuration)
-value to a suite or test, pass a configuration object to the test or suite
-function as the second argument.
+It is possible to apply [test-time configuration](/guides/references/configuration#Test-time-overrides) values during test execution.
```

Comment:
I would prefer to keep this section in this document. The configuration document is pretty overloaded as it is. Any reasoning for moving this section?

Reply:
I had removed this section since it repeated the explanation & examples snippet that are already provided in `/guides/references/configuration`. This is one less place to check for configuration documentation changes. I can re-add.